docs/services/data/_index.md

```diff
@@ -25,11 +25,11 @@ images: ["/services/icons/data-management.svg"]
 no_service: true
 ---
 
-The data management service is a robust solution for smart machine data handling, [data capture](/services/data/capture/), and [cloud synchronization](/services/data/cloud-sync/).
+The data management service is a robust solution for smart machine data handling, [data capture, and cloud synchronization](/services/data/capture-sync/).
```

Comment:
I don't like that the link is only on part of what's offered. I think then it's probably better not to have a link here. 
```suggestion
The data management service is a robust solution for handling machine data, allowing you to capture data, sync it to the cloud, and query it.
```